### PR TITLE
Ignore the immutable flatpak fwupd.conf file permissions

### PR DESCRIPTION
**Pull request by hughsie**
Fixes https://github.com/fwupd/fwupd/issues/5817

Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [X] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

File: libfwupdplugin/fu-config.c [hunk 1]
```diff
@@ -104,6 +104,8 @@ fu_config_reload(FuConfig *self, GError **error)
 	/* ensure the config files are set to the correct permissions */
 	for (guint i = 0; i < priv->filenames->len; i++) {
 		const gchar *fn = g_ptr_array_index(priv->filenames, i);
+		if (g_str_has_prefix(fn, "/app"))
```
Comment (superm1) [inline libfwupdplugin/fu-config.c:107]: We probably have the same problem for snap too? Maybe we should use environment variables instead.

Comment (hughsie) [reply libfwupdplugin/fu-config.c:107]: Can we ask the FS "is this immutable"? Is the chatter bit just set?

Comment (hughsie) [reply libfwupdplugin/fu-config.c:107]: Seems to work, will respin tomorrow.

Comment (hughsie) [reply libfwupdplugin/fu-config.c]: Done, all fixed up.

**Review by superm1 — APPROVED** [review]

**Pull request merged**